TrackletAlgorithm/VMRouterTop.h

```diff
@@ -2,46 +2,44 @@
 #define TrackletAlgorithm_VMRouterTop_h
 
 #include "VMRouter.h"
+#include "VMRouter_parameters.h"
 
-// VMRouter Top Function for layer 1, AllStub region E
+// VMRouter Top Function for layer 2, AllStub region A
 // Sort stubs into smaller regions in phi, i.e. Virtual Modules (VMs).
 
 // NOTE: to run a different phi region, change the following
 //          - constants specified in VMRouterTop.h
-//          - the input parameters to VMRouterTop in VMRouterTop.h/.cc
-//          - the the number and directories to the LUTs
-//          - add/remove pragmas depending on inputStub in VMRouterTop.cc
-//          - the call to VMRouter() in VMRouterTop.cc
-//          - the included top function in VMRouter_test.cpp (if file name is changed)
-//          - the top function and memory directory in script_VMR.tcl (if file name is changed)
+//          - add number to VMRouter_parameters.h if not already defined
 //          - add the phi region in emData/download.sh, make sure to also run clean
+//          - add/remove pragmas depending on inputStubs in VMRouterTop.cc (not necessary to run simulation)
 
 ////////////////////////////////////////////
 // Variables for that are specified with regards to the VMR region
 // Changed manually
 
-#define kLAYER 1 // Which barrel layer number the data is coming from
+#define kLAYER 2 // Which barrel layer number the data is coming from
 #define kDISK 0 // Which disk number the data is coming from, 0 if not disk
 
-constexpr char phiRegion = 'E'; // Which AllStub/PhiRegion
-constexpr int sector = 4; //  Specifies the sector
+constexpr phiRegions phiRegion = phiRegions::A; // Which AllStub/PhiRegion
 
-// Maximum number of memory "copies" for this Phi region
-// Note: can't use 0 if we don't have any memories of a certain type. Use 1.
-constexpr int maxASCopies(4); // Allstub memory
-constexpr int maxTEICopies(5); // TE Inner memories
-constexpr int maxOLCopies(3); // TE Inner Overlap memories
-constexpr int maxTEOCopies(1); // TE Outer memories
 
-// Number of inputs
-constexpr int numInputs(7); // Number of input memories, EXCLUDING DISK2S
-constexpr int numInputsDisk2S(0); // Number of DISK2S input memories
+///////////////////////////////////////////////
+// Variables that don't need manual changing
 
-constexpr int bendCutTableSize(8); // Number of entries in each bendcut table. Can't use 0.
+constexpr bool isLayer = (kLAYER) ? true : false;
```

Comment:
If the parameter in VMRouter_parameters.h is indeed redundant, then I think this can be removed.

Reply:
see reply above